src/xcode/ENA/HealthCertificateToolkit/Sources/HealthCertificateToolkit/CommonCovidLogic/CCLConfiguration.swift

```diff
@@ -42,3 +42,8 @@ public struct CCLConfiguration: Codable {
     public let validTo: String
     public let logic: Logic
 }
+
+public struct JsonFunctionDescriptor: Codable {
+    let name: String
+    let definition: JsonFunctionDefinition
+}
```

Comment:
That type already exists in JSONFunctions 😉

Reply:
I did not know. Then I will revert.